_rules/image-has-name-23a2a8.md

```diff
@@ -53,55 +53,64 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed Example 1
 
-The HTML `img` element has an accessible name
+The HTML `img` element has an accessible name given by the `alt` attribute.
```

Comment:
Arf. Me messing up with the image-has-name early merge is now causing it to appear in that PR where it shouldn't :-(

You need to revert that file to its previous version (the one that is now back on `develop`), I guess.
Sorry ☹️